src/material/button-toggle/button-toggle.html

```diff
@@ -1,8 +1,10 @@
 <button #button class="mat-button-toggle-button mat-focus-indicator"
         type="button"
         [id]="buttonId"
+        [attr.role]="_isSingleSelector() ? 'radio' : 'button'"
```

Comment:
Since we're overriding the role, don't we also have to set `aria-disabled`?

Reply:
Could you elaborate on why overriding the role has something to do with setting `aria-disabled` here? From my understanding button allows for the disabled attribute no matter what role is set.

Reply:
I think it'll stop being read out as disabled when the `role` is overwritten.

Reply:
Disabled attribute makes the screen reader skip the button when using the Tab key. Aria-disabled attribute instead allows users to tab to the button and then inform the screen reader that the button is disabled. Since toggle button already sets its disabled attribute, overriding the role of it will not change the fact that the button is not tabbable (Tested w/o this change), aria-disabled is not necessary in this case.

Reply:
I think that users can still discover the button even if they don't tab into it while they're navigating the page using some of the other screen reader shortcuts. This isn't a blocker, but it's also an easy fix to make.

Reply:
Additionally, setting both disabled and aria-disabled will not enable the button to be tabbable. To achieve the same disabled styling and functionality while keeping the button tabbable, it is necessary to not only replace the disabled property with use aria-disabled, but also to ensure that the button’s functionality is suppressed when it is in the disabled state. Aria-disabled only semantically exposes these elements as being disabled.

The aria-disabled attribute should ideally be used only when elements are required to be available for users to find when navigating via the Tab key.

Reply:
@crisbeto , that's right that there's some known limitations (or quirks) with disabled attribute. @clamli are going to cross browser test this on all supported screen readers before landing this.

Reply:
this looks resolved to me ✔️ . @clamli I'll follow up with you on manual testing before merging this.